diff --git a/Makefile b/Makefile
index 3cca0ca..bf70651 100644
--- a/Makefile
+++ b/Makefile
@@ -104,18 +104,7 @@ endif
 #Check GCC version so we can apply -Wno-date-time if supported.  GCC >= 4.9
 empty:=
 space:= $(empty) $(empty)
-GCCVERSIONSTRING := $(shell expr `$(CC) -dumpversion`)
-#Create version number without "."
-GCCVERSION := $(shell expr `echo $(GCCVERSIONSTRING)` | cut -f1 -d.)
-GCCVERSION += $(shell expr `echo $(GCCVERSIONSTRING)` | cut -f2 -d.)
-GCCVERSION += $(shell expr `echo $(GCCVERSIONSTRING)` | cut -f3 -d.)
-# Make sure the version number has at least 3 decimals
-GCCVERSION += 00
-# Remove spaces from the version number
-GCCVERSION := $(subst $(space),$(empty),$(GCCVERSION))
-# Crop the version number to 3 decimals.
-GCCVERSION := $(shell expr `echo $(GCCVERSION)` | cut -b1-3)
-GE_49 := $(shell expr `echo $(GCCVERSION)` \>= 490)
+GE_49 := 1

 ccflags-y :=

